loader/httploader/httploader.go

```diff
@@ -112,6 +113,10 @@ func (h *HTTPLoader) Get(r *http.Request, image string) (*imagor.Blob, error) {
 	return imagor.NewBlob(func() (io.ReadCloser, int64, error) {
 		resp, err := client.Do(req)
 		if err != nil {
+			if b, _ := regexp.MatchString("(no such host)$", err.Error()); b {
```

Comment:
What about `strings.Contains(err.Error(), "no such host")`, if it is just for simple string match?